test/files/run/t2318.scala

```diff
@@ -5,6 +5,7 @@ import scala.language.{ reflectiveCalls }
 object Test {
   trait Bar { def bar: Unit }
 
+  val forcePreloadBoxedUnit = scala.runtime.BoxedUnit.UNIT //Avoid parasiting ClassCircularityError, at least in Windows
```

Comment:
Is this related to your bugfix?

Reply:
It is unrelated.  Just an annoyance while testing with partest, that I addressed, for that it may help others.  Maybe I am one of the rare building on Windows.  Do you want me to extract it in a separate PR?

Reply:
No, that's fine, thanks